source/about/cloud-subscriptions.rst

```diff
@@ -73,6 +64,15 @@ Sales tax and VAT
 
 Mattermost reserves the right to assess applicable taxes as required by local law. Depending on location, you may be charged transaction taxes when purchasing our product. Prices on our website are exclusive of sales tax or VAT.
 
+Failed or late payments
+-----------------------
+
+You'll be notified immediately of failed payments both in-product, with a banner, and via email. The notifications provide directions on how to update payment information. Once payment information is updated, you'll be charged right away for the amount owing from the previous billing period.
+
+If you have not paid or resolved the failed payment after 60 days, your workspace will be downgraded to Mattermost Starter and subject to the data limits of that plan. Your workspace will also be suspended.
```

Comment:
```suggestion
If you have not paid or resolved the failed payment after 60 days, your workspace will be downgraded to Mattermost Starter and subject to the data limits of that plan. If your workspace is inactive for an additional 60 days after the downgrade, it will be deleted.
```